---
title: "&#39;&lt;name1&gt;&#39;이 &#39;&lt;name2&gt;&#39;에서 상속되므로 &#39;Catch&#39; 블록에 도달하지 않습니다. | Microsoft Docs"
ms.custom: ""
ms.date: "11/16/2016"
ms.prod: "visual-studio-dev14"
ms.reviewer: ""
ms.suite: ""
ms.technology: 
  - "devlang-visual-basic"
ms.tgt_pltfrm: ""
ms.topic: "article"
f1_keywords: 
  - "BC42029"
  - "vbc42029"
helpviewer_keywords: 
  - "BC42029"
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
caps.latest.revision: 12
caps.handback.revision: 12
author: "stevehoag"
ms.author: "shoag"
manager: "wpickett"
---
# &#39;&lt;name1&gt;&#39;이 &#39;&lt;name2&gt;&#39;에서 상속되므로 &#39;Catch&#39; 블록에 도달하지 않습니다.
코드 내의 `Catch` 블록에 도달하지 않습니다.  
  
 이 메시지는 기본적으로 경고입니다. 경고를 숨기거나 오류로 처리하는 방법에 대한 자세한 내용은 [Visual Basic에서 경고 구성](../Topic/Configuring%20Warnings%20in%20Visual%20Basic.md)을 참조하세요.  
  
 **오류 ID:** BC42029  
  
### 이 오류를 해결하려면  
  
-   불필요한 `Catch` 블록을 제거합니다.  
  
## 참고 항목  
 [방법: Visual Basic에서 Try…Catch 블록을 사용하여 코드 테스트](http://msdn.microsoft.com/ko-kr/8368e205-ed73-4185-a247-af84fb4fafa9)   
 [Try...Catch...Finally Statement](../Topic/Try...Catch...Finally%20Statement%20\(Visual%20Basic\).md)